### PR TITLE
password reset

### PR DESCRIPTION
**Pull request by canhchimmcodon**

### CODE REVIEW

File: app/views/password_resets/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<% provide(:title, t(".reset_password")) %>
```
Comment (anhnt796) [inline app/views/password_resets/edit.html.erb]: Xóa ()

File: app/views/password_resets/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<% provide(:title, t(".forgot_password")) %>
```
Comment (anhnt796) [inline app/views/password_resets/new.html.erb]: Xóa ()

File: app/views/password_resets/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+<h1><%= t ".forgot_password" %></h1>
+
+<div class="row">
+  <div class="col-md-6">
+    <%= form_for(:password_reset, url: password_resets_path) do |f| %>
```
Comment (anhnt796) [inline app/views/password_resets/new.html.erb]: Xóa ()

File: app/views/user_mailer/password_reset.html.erb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<h1><%= t ".pasres" %></h1>
+<p><%= t ".text1" %></p>
+<%= link_to t(".reset_pass"), 
+  edit_password_reset_url(@user.reset_token, email: @user.email)%>
```
Comment (anhnt796) [inline app/views/user_mailer/password_reset.html.erb]: Xóa ()

**Comment by canhchimmcodon** [comment]
ready

**Comment by baclv** [comment]
merged

**Pull request merged**